### PR TITLE
fix(lb): rename loadbalancer

### PR DESCRIPTION
**Pull request by Monitob**
- fix(lb): rename test data source load balancer [error](https://github.com/scaleway/terraform-provider-scaleway/runs/4389230988?check_suite_focus=true#step:4:11862)

### CODE REVIEW

File: scaleway/data_source_lb_test.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ func TestAccScalewayDataSourceLb_Basic(t *testing.T) {
 
 					resource scaleway_lb main {
 					    ip_id = scaleway_lb_ip.main.id
-						name = "test-lb"
+						name = "data-test-lb"
```
Comment (remyleone) [inline scaleway/data_source_lb_test.go:24]: What does this test fix?

Comment (Monitob) [reply scaleway/data_source_lb_test.go:24]: It's a  terraform error: `Error: 2 lbs found with the same name test-lb` when launching in terraform apply in parallel with resources. There is no error from our apis.

Comment (remyleone) [reply scaleway/data_source_lb_test.go:24]: It is really strange. Could you add a test that it is legit to have two lb with the same name? Normally they are made unique by their id, not their name.

Comment (remyleone) [reply scaleway/data_source_lb_test.go:24]: I think we can keep the name it removes a line from the diff :)

**Review by remyleone — APPROVED** [review]

**Pull request merged**